### PR TITLE
Heart failure artifact

### PR DESCRIPTION
**Pull request by hussain-jafari**
### Description
- *Category*: data artifact
- *JIRA issue*: [MIC-3355](https://jira.ihme.washington.edu/browse/MIC-3355)
- *Research reference*: [Cause Model Diagram to Data Sources](https://vivarium-research.readthedocs.io/en/latest/models/causes/ischemic_heart_disease/gbd_2019/index.html?highlight=heart%20failure#vivarium-modeling-strategy)

### Verification and Testing
Loaded data in notebook and briefly checked they looked reasonable and generated artifact.

### CODE REVIEW

File: src/vivarium_nih_us_cvd/data/loader.py
Comment (stevebachmeier) [inline src/vivarium_nih_us_cvd/data/loader.py:312]: List[str]. I know that was just a copy/paste but seems incorrect (unless it's just a way to better type hint? Like...it's just just a string but a string of sequela?)

File: src/vivarium_nih_us_cvd/data/loader.py
Comment (stevebachmeier) [inline src/vivarium_nih_us_cvd/data/loader.py:414]: Don't think you need this since you do it again anyway a few lines below after multiplying

File: src/vivarium_nih_us_cvd/data/loader.py
Comment (stevebachmeier) [reply src/vivarium_nih_us_cvd/data/loader.py:312]: Eh, I'd just leave it - it happens a lot here. I must've copied/pasted old code so it might just be a way of more specifically specifying what type of string the list will consist of.

File: src/vivarium_nih_us_cvd/data/loader.py
Comment (stevebachmeier) [inline src/vivarium_nih_us_cvd/data/loader.py:460]: nit: the tuple unpacking threw me. Consider just `if key == data_keys.IHD_AND_HF.PREVALENCE_ACUTE_MI_AND_HF`

File: src/vivarium_nih_us_cvd/data/loader.py
Comment (stevebachmeier) [inline src/vivarium_nih_us_cvd/data/loader.py:463]: So the sum of these three prevalences is the prevalence of the full envelope?

File: src/vivarium_nih_us_cvd/data/loader.py
Comment (stevebachmeier) [inline src/vivarium_nih_us_cvd/data/loader.py:585]: I think yes but confirm w/ Rajan

File: src/vivarium_nih_us_cvd/data/loader.py
Comment (stevebachmeier) [inline src/vivarium_nih_us_cvd/data/loader.py:93]: I'm not seeing the equations for hf_ihd and residual disability weights in the docs. I assume these come from discussion you have with Syl? Be sure those docs get updated if so

**Review by stevebachmeier — APPROVED** [review]

**Comment by rmudambi** [comment]
I have a question for Syl about the heart failure model. I'm going to hold off on finishing this review until I see the answer.

File: src/vivarium_nih_us_cvd/data/loader.py
Comment (hussain-jafari) [reply src/vivarium_nih_us_cvd/data/loader.py:312]: They're actually Sequela objects and not strs so it might actually be appropriate to remove the quotes.

File: src/vivarium_nih_us_cvd/data/loader.py
Comment (hussain-jafari) [reply src/vivarium_nih_us_cvd/data/loader.py:414]: The second time is to remove the proportion column that got added after merging the proportion data.

File: src/vivarium_nih_us_cvd/data/loader.py
Comment (hussain-jafari) [reply src/vivarium_nih_us_cvd/data/loader.py:460]: I find including prevalence_includes_heart_failure more readable but I'm open to what other reviewers say.

File: src/vivarium_nih_us_cvd/data/loader.py
Comment (hussain-jafari) [reply src/vivarium_nih_us_cvd/data/loader.py:463]: Yeah I talked to Syl that this what we wanted since I was also concerned about this. It's a bit tricky because if you experience acute myocardial infarction AND heart failure, your "post MI" state is actually heart failure from ischemic heart disease.

File: src/vivarium_nih_us_cvd/data/loader.py
Comment (hussain-jafari) [reply src/vivarium_nih_us_cvd/data/loader.py:93]: I found them [here](https://vivarium-research.readthedocs.io/en/latest/models/causes/ischemic_heart_disease/gbd_2019/index.html#states-data) in the last entry and fourth to last entry for residual heart failure and heart failure from ischemic heart disease respectively.

**Review by rmudambi — APPROVED** [review]

**Pull request merged**
